index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="eng">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <Title>Portfolio</Title>
+        <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+
+    <body>
+      <header>
+        <nav class="header">
+          <h2 class="my-logo">My Logo</h2>
+          <a href="#"><img src="./logos/Menu.png" alt="Menu"></a>
```

Comment:
- [ ] As mentioned by the previous reviewer and for best semantics tags, I think it would be better if you use another tag than `a`(for example to wrap the `img` tag inside a  `div` instead of a `a` tag) on your hamburger button instead you should use it on your logo text i.e your logo text should be a link.